pkg/provider/operators.go

```diff
@@ -70,6 +70,11 @@ func (op *Operator) String() string {
 
 //nolint:funlen
 func (op *Operator) SetPreflightResults(env *TestEnvironment) error {
+	if len(op.InstallPlans) == 0 {
+		logrus.Warnf("%s has no InstallPlans. Skipping setting preflight results", op.String())
+		return nil
```

Comment:
I'm not sure if I want to actually throw an error here, but rather just skip the operator and exclude it from the preflight tests at this point.  It's open to debate.